go/teams/service_helper.go

```diff
@@ -1245,61 +1247,84 @@ func CanUserPerform(ctx context.Context, g *libkb.GlobalContext, teamname string
 		return isRoleOrAbove(keybase1.TeamRole_WRITER)
 	}
 
-	isAdminOrImplicitAdmin := func() (bool, error) {
-		ret, err := isAdmin()
+	canMemberShowcase := func() (bool, error) {
 		if err != nil {
 			return false, err
 		}
-		if ret {
+		if teamRole.IsOrAbove(keybase1.TeamRole_ADMIN) {
 			return true, nil
+		} else if teamRole == keybase1.TeamRole_NONE {
+			return false, nil
 		}
-		return isImplicitAdmin()
+		showcase, err := GetTeamShowcase(ctx, g, teamname)
+		if err != nil {
+			return false, err
+		}
+		return showcase.AnyMemberShowcase, nil
 	}
 
-	canMemberShowcase := func() (bool, error) {
-		r, err := team.MemberRole(ctx, meUV)
+	hasOtherOwner := func() (bool, error) {
+
+		t, err := GetForTeamManagementByStringName(ctx, g, teamname, true)
```

Comment:
Github diff gets really nasty here so maybe I'm missing something, but - can we use `team` object that we already have instead of loading team again?

Reply:
The guiding light should be to make as little team load calls as possible in this function.